app/assets/sass/application-common.scss

```diff
@@ -17,69 +17,65 @@
 }
 
 // Add extra styles here, or re-organise the Sass files in whichever way makes most sense to you
- .button-secondary{
-   @include button($grey-3);
+.button-secondary{
+  @include button($grey-3);
 }
 
 // Tasklist page start
-  li.task-item {
-    list-style: decimal inside none;
-    border-top: 1px solid $grey-2;
-    line-height: 1.5;
-    padding: 20px 0 10px 0;
-    font-weight: bold;
-
-    @include media(tablet) {
-      font-size: 22px;
-      list-style-position: outside;
-    }
+li.task-item {
+  border-top: 1px solid $grey-2;
+  line-height: 1.5;
+  padding: 20px 0 10px 0;
 
-    .task-item-header {
-      display: inline;
+  .task-item-header {
+    overflow: hidden;
 
-      .heading-medium {
-        display: inline;
+    .heading-medium {
+      margin-bottom: 0;
 
-        @include media(tablet) {
-          display: inline-table;
-          width: 66.6%;
-          margin-top: 0;
-        }
-      }
-      .task-unavailable {
-        background-color: $grey-2;
-        color: $black;
-        float: right;
+      @include media(tablet) {
+        float: left;
+        width: 66.6%;
```

Comment:
The GDS breakpoint variables are:

```
$tablet-breakpoint: 641px !default;
$desktop-breakpoint: 769px !default;
```

A) Ideally we should have the title and tag on the same line for the desktop size, at the moment it appears to break before the browser is reduced to this size at around `785px`. It appears that reducing the title width to `65%` would fix this...is `66.6%` from the GDS styling or a width you came up with?

For example, from this:

![screen shot 2018-06-29 at 12 09 51](https://user-images.githubusercontent.com/6839214/42089540-7fffe52c-7b95-11e8-9616-f1b0b0542515.png)

To this:

![screen shot 2018-06-29 at 12 10 16](https://user-images.githubusercontent.com/6839214/42089548-888e4b20-7b95-11e8-9caa-faf77f9ee69e.png)


  Can you also check that the title is fully indented so it doesn't appear as in the second example above.

B) When the tag breaks to the next line for tablet screen sizes and smaller I think the following should happen:

- The title becomes full with.
- The top and bottom margin around the tag increases, perhaps by `20px`.
- We should consider aligning the tag to the left.

For example, from this:

![screen shot 2018-06-29 at 11 56 08](https://user-images.githubusercontent.com/6839214/42089041-9deba29e-7b93-11e8-997f-87387a5469f3.png)

To this:

![screen shot 2018-06-29 at 11 56 49](https://user-images.githubusercontent.com/6839214/42089049-a6fb2206-7b93-11e8-8ec9-c37fe7d7764c.png)

And this:

![screen shot 2018-06-29 at 11 40 07](https://user-images.githubusercontent.com/6839214/42088910-2501b562-7b93-11e8-9305-a3aca68e1f11.png)

To this:

![screen shot 2018-06-29 at 11 40 55](https://user-images.githubusercontent.com/6839214/42088890-0a9a35d2-7b93-11e8-9013-6876c62322f9.png)